components/sbm-openrewrite/src/test/java/org/springframework/sbm/openrewrite/maven/BumpParentVersionTest.java

```diff
@@ -61,7 +61,8 @@ void bumpedParentPomVersionIsNotReflectedInModelTest() {
 
         List<Xml.Document> parsedPom = MavenParser.builder().build().parse(pomXml);
 
-        List<Result> results = new UpgradeParentVersion("org.springframework.boot", "spring-boot-starter-parent", "3.0.0-M3", null).run(parsedPom);
+        List<Result> results = new UpgradeParentVersion("org.springframework.boot", "spring-boot-starter-parent", "2.7.1", null).run(parsedPom);
```

Comment:
The test was correct. Now it's setting the version of the parent pom to the version it already has and the `UpgradeParentPom` recipe will do nothing.
The issue (before upgrading OpenRewrite to a more recent version as on `main` now) was that the change to `3.0.0-M3` was applied in the `pom.xml` source (given that the milestone repo was configured)  but not reflected in the pom's model. 
This is the case now in `main` and the test proved that in latest OR version the change is reflected in the model too.
The comment should be removed as the call to `RefreshPomModel` in `OpenRewriteMavenBuildFile.upgradeParentVersion()` has been removed.

Reply:
Hm. Sounds like the root problem was that Rewrite couldn't retrieve the parent in `3.0.0-M3` from the configured milestone repo?
Actually, the test could be deleted. I'd say. 😬 
After removing the call to `RefreshPomModel` in `OpenRewriteMavenBuildFile.upgradeParentVersion()` after upgrading rewrite to  `7.26.0` changes to the parent version are reflected in the underlying model by rewrite automatically. The intention of the test was to remind us about this fact.